CCP_App/docker-compose.yaml

```diff
@@ -0,0 +1,25 @@
+version: '3.7'
+
+services:
+
+  mongodb:
+    container_name: mongodb
+    image: mongo
+    ports:
+      - "27017:27017"
+
+  claimsapi:
+    container_name: springTestApi
+    image: springapi:latest
+    environment:
+      spring_data_mongodb_host: mongodb
+    ports:
+      - "8080:8080"
+    depends_on:
+      - mongodb
+  rabbitmq3:
+    image: rabbitmq:3-management
+    environment:
+      RABBITMQ_ERLANG_COOKIE: 'secret cookie here'
```

Comment:
consider removing the RABBITMQ_ERLANG_COOKIE environment variable. At best its doing nothing. At worst it might cause issues later on. Alternatively read the docs and see what this is used for.